src/Npgsql/NpgsqlConnectionStringBuilder.cs

```diff
@@ -503,6 +503,44 @@ public string Encoding
         }
         string? _timezone;
 
+        /// <summary>
+        /// Determines the preferred PostgreSQL target server type.
+        /// </summary>
+        [Category("Connection")]
+        [Description("Determines the preferred PostgreSQL target server type.")]
+        [DisplayName("TargetSessionAttributes")]
+        [DefaultValue(TargetSessionAttributes.Any)]
```

Comment:
Just converting my test console apps to use this build; I couldn't find this without looking at the diff and I don't think I would have guessed the name. Is there reason not to maintain parity with the JDBC driver? https://jdbc.postgresql.org/documentation/head/connect.html#connection-parameters (they use `TargetServerType`)

Reply:
@roji thought it would be better to mirror the [libpq](https://www.postgresql.org/docs/devel/libpq-connect.html). I personally do not have a strong preference to either.

Reply:
I'll be doing some documentation on this to make it discoverable... Note that we're also following libpq on the contents of that settings - we support ReadWrite/ReadOnly as well as Primary/Standby, unlike JDBC.